src/utstring.h

```diff
@@ -82,8 +82,8 @@ do {                                                       \
 #define utstring_new(s)                                    \
 do {                                                       \
    (s) = (UT_string*)malloc(sizeof(UT_string));            \
-   if (!(s)) oom();                                        \
-   utstring_init(s);                                       \
+   if ((s)) utstring_init(s);                              \
```

Comment:
Redundant parens in `if ((s))` — should be just `if (s)`.

Reply:
Hm... I've followed the same used in https://github.com/troydhanson/uthash/blob/master/src/utstring.h#L54, https://github.com/troydhanson/uthash/blob/master/src/utstring.h#L85.

Reply:
FYI, the difference there is that those single sets of parens are actually doing something. There's no instance of `((s))` in what you linked to — just `(s)`.

Reply:
Interesting information. I'm going to fix it to upgrade the PR ...

Reply:
Removed the redundant parenthesis in `if`.